### ISSUE TITLE
Make micButton target more resilient

### ISSUE THREAD

**Issue by andybraren** [andybraren/google-meet-fixer#5]
The mic toggle button is currently targeted using aria attributes and platform-specific text, like `aria-label="Turn off microphone (⌘ + d)"`. That's probably not the most resilient or platform-agnostic way to target that button, so something else should be used instead.
